tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerApiParser.csproj

```diff
@@ -6,10 +6,11 @@
     <ToolCommandName>swaggerAPIParser</ToolCommandName>
     <TargetFramework>net6.0</TargetFramework>
     <RootNamespace>swagger_api_parser</RootNamespace>
-    <VersionPrefix>1.0.5</VersionPrefix>
+    <VersionPrefix>1.0.6</VersionPrefix>
     <AssemblyName>Azure.Sdk.Tools.SwaggerApiParser</AssemblyName>
     <SignAssembly>True</SignAssembly>
     <InformationalVersion>$(OfficialBuildId)</InformationalVersion>
+    <LangVersion>10</LangVersion>
```

Comment:
Any particular reason we need to override the default version here?

Reply:
There are some features need to specify c# version. Like https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces

Reply:
Based on what I'm reading at https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version#defaults I would expect since you are targeting .NET 6 that would be the default language version. However not a big deal either way I just like to avoid having these be explicitly defined in the project if they aren't needed.

Reply:
I remove the hard-coded language version. Looks build and test succeed.